source/agora/consensus/protocol/Nominator.d

```diff
@@ -229,6 +235,8 @@ extern(D):
         this.timers[TimersIdx.Envelope] = this.taskman.createTimer(&this.envelopeProcessTask);
         this.timers[TimersIdx.Nomination] = this.taskman.createTimer(&this.checkNominate);
         this.timers[TimersIdx.Catchup] = catchup_timer; // Created in caller
+        this.slot_stat.clearStats();
```

Comment:
Why do we need a clear here ?

Reply:
We dont need it there but after restoring SCP state.